src/Features/LanguageServer/Protocol/Handler/Completion/CompletionResolveData.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 {
     internal class CompletionResolveData
     {
-        public CompletionParams CompletionParams { get; set; }
-        public string DisplayText { get; set; }
+        public CompletionParams? CompletionParams { get; set; }
+        public string? DisplayText { get; set; }
```

Comment:
Why nullable? Can we avoid it if we add a constructor to this?

Reply:
Also used for serialization.